modules/ROOT/pages/admin-center.adoc

```diff
@@ -128,14 +128,33 @@ You can add more resource metrics to the **Monitor** view by selecting the **Edi
 [#batch]
 === Manage Java batch jobs with the Java Batch tool
 
-If you configure the feature:batchManagement[display=Batch Management] feature, you can access the **Java Batch** tool. With this tool, you can view the progress and status of your Java batch jobs, manage their instances, and view their log files. The following screen capture shows the **Java Batch** tool:
+If you configure�the feature:batchManagement[display=Batch Management] feature, you can access the **Java Batch** tool. With this tool, you can view the progress and status of your Java batch jobs, manage their instances, and view their log files. The following screen capture shows the **Java Batch** tool:
 
 image::ui_javaBatchTool.png[align="center"]
 
 Each batch job has an **Actions** icon, which you can select to stop, restart, or purge the job, and a **View Logs** icon, which you can select to view the job logs. To view details and metrics for a batch job, hover over the **Batch Job Name** and select an option from the tooltip menu.
 
 If batch jobs or job logs are on remote servers, link:/guides/cors.html[configure cross origin region sharing (CORS)] on each remote server. CORS enables Admin Center to request job information from remote servers.
 
+Note too that the batch features use a custom authorization specific to batch and role membership must be granted separately from the roles needed to access the Admin Center in the first place.  
+
+E.g. a 'batchadmin' user configured like:
+
+[source,xml]
+----
+  <reader-role>
```

Comment:
i think it's potentially confusing in this example to have one role called `batchadmin` and another called `batchAdmin`. If the security-role requires `batchAdmin` can we rename the reader role to something like `batchToolUser` or `wanda`?

Reply:
It's one user, batchadmin, added to each of two different roles:  reader-role and batchAdmin.  The user needs both roles.

Reply:
Yes, I know its the same user, but i think potential for confusion is between the user name `batchadmin` and the role name `batchAdmin`. I was just thinking if either of those example IDs is arbitrary, then making them a little more different would help disambiguate their purpose in the example. 

As is, a reader might not catch the minor difference in the two strings and wonder if they need to match the role to the ID in their environment (Spoiler alert: this happened to me :)

Not a major concern- if the config requires both names this way, or if you just disagree, I'm ok with it as is.

Reply:
It's one user, batchadmin, added to each of two different roles: reader-role and batchAdmin. The user needs both roles.

Reply:
OK, sure, good point.     Sorry.. I wasn't trying to be stubborn I just somehow left my comment twice :)